### ISSUE TITLE
 Autodoc errors for documentation of tests/test_dbf

### ISSUE THREAD

**Issue by Jack-Hawthorne** [architecture-building-systems/CityEnergyAnalyst#1731]
Getting the following issues with autodoc for cea/tests/test_dbf.py :

```
WARNING: autodoc: failed to import module u'test_dbf' from module u'cea.tests'; the following exception was raised:
Traceback (most recent call last):
  File "C:\Users\Jack\Miniconda2\envs\cea\lib\site-packages\sphinx\ext\autodoc\importer.py", line 154, in import_module
    __import__(modname)
  File "c:\users\jack\documents\github\cityenergyanalyst\cea\tests\test_dbf.py", line 8, in <module>
    from pandas.testing import assert_frame_equal
RuntimeError: sys.path must be a list of directory names
```

This should be an easy fix.
